flow_screen_components/quickChoiceFSCproject/force-app/main/default/lwc/quickChoiceFSC/quickChoiceFSC.js

```diff
@@ -51,7 +53,8 @@ export default class SmartChoiceFSC extends LightningElement {
 	//possibility master record type only works if there aren't other record types?
 	@wire(getPicklistValues, {
 		recordTypeId: "$recordTypeId",
-		fieldApiName: "$calculatedObjectAndFieldName"
+		fieldApiName: "$objectAndFieldName"
```

Comment:
It looks like this code is flipflopping. I originally wanted to do separate object and field attributes because they're simpler to explain, but in the very first version I hadn't figured out how to use the getters and setters to properly generate a calculated version that could be used (the calculated version is then used by the getPicklistValues wire service. Once I figured that out, I removed the "ObjectAndFieldName" attribute and don't want to re-add it, as is proposed in this commit.

I suggest that you cancel this commit, pull from the current codeline (at the moment the latest commit is Feb1 https://github.com/alexed1/LightningFlowComponents/commit/efb3a491e5dbb97862202b9da80a0c05b610b39c), add your rendered callback fix, take the fix that @clifford-fra suggests, if you agree with it, and create a new pull request with just those two fixes.

Reply:
Alex - I have the updates made locally but I'm not able to get them to the managed package dev org because I can delete the older versions of the test flow.  Do you have a way of removing the older flow version?

Reply:
I looked into it a bit and couldn't find a way. deleting stuff from managed packages is deliberately hard to do. all you can really do is deprecate stuff and ask the user to manually delete it. So I think we should just abandon that hosting org and create a new one. why don't we divide and conquer: go put the latest version here: 

hostquick@deved.org | test1234

I'm on the fence about whether to go managed or unmanaged. This component is getting a fair amount of updates so maybe we leave it unmanaged for now. But if we decide to, we can give hostquick a namespace and generate a managed package

Reply:
Please email me the Verification Code for logging in to the new org